### DISCUSSION TITLE
Clarify build requirements in README

### DISCUSSION THREAD

**Discussion by ma11hew28** [protobuf-c/protobuf-c#599] [Q&A]
The README says protobuf-c requires [protobuf][1] to be installed.

Do I need to install the protocol compiler (`protoc`), the protobuf runtime, or both?

If I need to install the protocol compiler, do I also need to install "the set of standard .proto files" that come with my platform's pre-built `protoc` binary (published on the [protobuf releases][2] page)?

If I need to install the protobuf runtime, then for which programming language?

Also, perhaps we should update the protobuf link address to https://github.com/protocolbuffers/protobuf because https://github.com/google/protobuf permanently (301) redirects to that address.

  [1]: https://github.com/protocolbuffers/protobuf
  [2]: https://github.com/protocolbuffers/protobuf/releases

**Comment by lipnitsk** [comment]
I think it depends on your use case. If you have a better wording suggestion, please open a pull request and we will review it.
